testsuite/db/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything
```

Comment:
Do we not need the db directory anymore?

Reply:
If this directory is absent, I think our db init fails. (But if that's wrong, I'd love to know and let's kill it.)

Reply:
@jmandel is correct.